src/DIRAC/Core/DISET/private/FileHelper.py

```diff
@@ -145,7 +145,7 @@ def errorInTransmission(self):
     def networkToString(self, maxFileSize=0):
         """Receive the input from a DISET client and return it as a string"""
 
-        stringIO = StringIO()
+        stringIO = BytesIO()
```

Comment:
```suggestion
        bytesIO = BytesIO()
```
I would propose to rename the variable accordingly.

Reply:
Thank you for your suggestion!